sem_covid/entrypoints/notebooks/topic_modeling/topic_modeling_wrangling/tfidf_vectorization.py

```diff
@@ -0,0 +1,23 @@
+
+from typing import List
+
+import spacy
+import pandas as pd
+
+from sklearn.feature_extraction.text import TfidfVectorizer
+
+nlp = spacy.load("en_core_web_sm")
+stopwords = nlp.Defaults.stop_words
+
+
+def vectorize_documents(document: List) -> pd.DataFrame:
```

Comment:
Don't we already have this defined elsewhere? 

Please standardise the operations and place them in a single place. Most importantly, reuse these operations.

Reply:
yes, I found it. I'll replace the function